Source/EVEClassLibrary/EVEClassLibrary/ConsoleMain.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Runtime;
+using System.IO;
+using Newtonsoft;
+
+namespace EVEClassLibrary
+{
+    /// <summary>
+    /// Main Console Class, Handles the Basic User Input
+    /// </summary>
+    public class ConsoleMain
+    {
+        /// <summary>
+        /// Initializing Console
+        /// </summary>
+        public static void init()
+        {
```

Comment:
Consider making this a loop

Reply:
will be changed at a later date

Reply:
I'll create an issue for that so that this comment can be closed